lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -96,6 +102,24 @@ public function getObjectManager()
     /** @inheritDoc */
     public function purge()
     {
+        $connection = $this->em->getConnection();
+
+        if ($this->cachedSqlStatements === null) {
```

Comment:
I would put this handling of the cache in the private method instead:

```php
public function purge()
{
    array_map([$connection, 'executeStatement'], $this->getPurgeStatements());
}

private function getPurgeStatements(): array
{
    if ($this->cachedSqlStatements !== null) {
        return $this->cachedSqlStatements;
    }

    // ...

    return $this->cachedSqlStatements = $statements;
}
```

Reply:
done